R/help.R

```diff
@@ -66,6 +66,51 @@ register_module_help_handler <- function(module, handler) {
   .module_help_handlers[[module]] <- handler
 }
 
+# Extract docs in TensorFlow-like styles
+help_completion_handler_default <- function(doc) {
+  arguments_matches <- regexpr(pattern = '\n(Arg(s|uments):)', doc)
+  if (arguments_matches[[1]] != -1)
```

Comment:
Just curious why you use `[[1]]` indexing here? From what I understand, `regexpr()` only returns the first match, and so the return value should always be a length-one vector (although with some extra attributes). However, those attributes won't be used during comparison with `==` or `!=`.

I think the existing code is still correct but curious if there is some other motivation.